privval/dash_core_signer_client.go

```
@@ -176,21 +194,25 @@ func (sc *DashCoreSignerClient) GetProTxHash() (crypto.ProTxHash, error) {
 	}
 	if len(decodedProTxHash) != crypto.DefaultHashSize {
 		// We are proof of service banned. Get the proTxHash from our IP Address
-		networkInfo, err := sc.dashCoreRpcClient.GetNetworkInfo()
+		networkInfo, err := sc.dashCoreRPCClient.GetNetworkInfo()
 		if err == nil && len(networkInfo.LocalAddresses) > 0 {
 			localAddress := networkInfo.LocalAddresses[0].Address
 			localPort := networkInfo.LocalAddresses[0].Port
 			localHost := fmt.Sprintf("%s:%d", localAddress, localPort)
-			results, err := sc.dashCoreRpcClient.MasternodeListJSON(localHost)
+			results, err := sc.dashCoreRPCClient.MasternodeListJSON(localHost)
 			if err == nil {
 				for _, v := range results {
-					decodedProTxHash, err = hex.DecodeString(v.ProTxHash)
+					decodedProTxHash, _ = hex.DecodeString(v.ProTxHash)
```

you can't ignore err here.